custom_components/ithodaalderop/definitions.py

```diff
@@ -181,14 +181,55 @@ class IthoBinarySensorEntityDescription(BinarySensorEntityDescription):
     ),
 )
 
+CVEBINARYSENSORS: tuple[IthoBinarySensorEntityDescription, ...] = (
+    IthoBinarySensorEntityDescription(
+        json_field="Filter dirty",
+        key=MQTT_STATETOPIC["hru"],
+        translation_key="filter_dirty",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+        icon_off="mdi:air-filter",
+        icon_on="mdi:vacuum-outline",
+    ),
+)
+
 CVESENSORS: tuple[IthoSensorEntityDescription, ...] = (
+    IthoSensorEntityDescription(
+        json_field="Error",
+        key=MQTT_STATETOPIC["hru"],
+        translation_key="error",
+    ),
     IthoSensorEntityDescription(
         json_field="Fan setpoint (rpm)",
         key=MQTT_STATETOPIC["hru"],
         translation_key="fan_setpoint_rpm",
         native_unit_of_measurement=REVOLUTIONS_PER_MINUTE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    IthoSensorEntityDescription(
+        json_field="Fan speed (rpm)",
+        key=MQTT_STATETOPIC["hru"],
+        translation_key="fan_speed",
+        native_unit_of_measurement=REVOLUTIONS_PER_MINUTE,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=False,
+    ),
+    IthoSensorEntityDescription(
+        json_field="Highest CO2 concentration (ppm)",
+        key=MQTT_STATETOPIC["hru"],
+        translation_key="highest_received_co2_value",
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=False,
+        icon="mdi:molecule-co2",
+    ),
+    IthoSensorEntityDescription(
+        json_field="Highest RH concentration (%)",
+        key=MQTT_STATETOPIC["hru"],
+        translation_key="highest_received_rh_value",
+        native_unit_of_measurement=PERCENTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=False,
+    ),
     IthoSensorEntityDescription(
         json_field="hum",
```

Comment:
@jasperslits I highly doubt this json_field is correct. Can you verify this somehow?

Reply:
Same for "temp"

Reply:
Thanks. The naming of these is just so different compared to the rest